app/assets/stylesheets/components/_list.scss

```diff
@@ -23,40 +42,14 @@
   }
 }
 
-.teal-dots {
-  list-style: none;
-  padding: 0;
-
-  li {
-    padding-left: 1rem;
-    text-indent: -1rem;
+.red-dots {
+  @include color-list($red)
+}
 
-    &::before {
-      color: $teal;
-      content: '•';
-      font-size: 2rem;
-      line-height: 1.5rem;
-      padding-right: .5rem;
-      vertical-align: -3px;
-    }
-  }
+.teal-dots {
+  @include color-list($teal)
 }
 
 .yellow-dots {
-  list-style: none;
-  padding: 0;
-
-  li {
-    padding-left: 1rem;
-    text-indent: -1rem;
-
-    &::before {
-      color: $yellow;
-      content: '•';
-      font-size: 2rem;
-      line-height: 1.5rem;
-      padding-right: .5rem;
-      vertical-align: -3px;
-    }
-  }
+  @include color-list();
```

Comment:
@jgsmith-usds Taking advantage of default parameters for the mixin (but don't need to).

Reply:
I'm a lot more comfortable specifying the color here since the goal is yellow dots. If the default ever changed, the color wouldn't match. Now, if we had `.default-dots`, then I'd probably want to not have an explicit color. :-)